service.md

```diff
@@ -56,24 +100,47 @@ Connect to the VirtualMachine by using a node IP and node port outside the clust
 $ ssh cirros@$NODE_IP -p 30000
 ```
 
-### Expose VirtualMachine as a single port service
+### Expose VirtualMachine as a LoadBalancer service
 
-Create a single port `service` object by using `kubectl`:
+Expose the RDP port (3389) of a VirtualMachine running on KubeVirt by creating `LoadBalancer service`. Here is an example of a LoadBalancer service:
 
-```bash
-$ kubectl expose pod virt-launcher-testvm-ephemeral-9bqv4 --port=27017 --target-port=22 --name=vmservice
+```
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    kubevirt.io: virt-launcher
+    kubevirt.io/domain: windows2012r2
+  name: lbsvc
+spec:
+  externalTrafficPolicy: Cluster
+  ports:
+  - port: 27017
+    protocol: TCP
+    targetPort: 3389
+  selector:
+    kubevirt.io: virt-launcher
+    kubevirt.io/domain: windows2012r2
+  type: LoadBalancer
 ```
 
-Query the service object:
+You could create this `LoadBalancer` service by using `kubectl`:
 
 ```bash
-$ kubectl get service
-NAME        TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)     AGE
-vmservice   ClusterIP   172.30.3.149   <none>        27017/TCP   2m
+$ kubectl -f lbsvc.yaml
+# OR
+$ kubectl get pod
+NAME                                READY     STATUS    RESTARTS   AGE
+virt-launcher-windows2012r2-x9x2f   1/1       Running   0          9m
+$ kubectl expose pod virt-launcher-windows2012r2-x9x2f --port=27017 --target-port=3389 --name=lbsvc --type=LoadBalancer
```

Comment:
If the `--type=LoadBalancer` also behaves like nodePort on k8s 1.9 I am fine  with this. @duyanyan is that the case?

Reply:
Yes, I think so.